apps/prairielearn/src/pages/authCallbackOAuth2/authCallbackOAuth2.ts

```diff
@@ -27,6 +27,7 @@ router.get(
     if (code == null) {
       throw new Error('No "code" query parameter for authCallbackOAuth2');
     } else if (typeof code !== 'string') {
+      // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
```

Comment:
Nit, would an explicit `toString()` call be better?

Reply:
Done in ed32c0fd05e656102949ea224dc4336fbd3fba0f.